Project.toml

```diff
@@ -55,7 +55,6 @@ LinearAlgebra = "1"
 NCDatasets = "0.11, 0.12, 0.13, 0.14"
 SciMLBase = "1, 2"
 StaticArrays = "1"
-StatsBase = "0.34"
```

Comment:
I think the compat should be `StatsBase = "0.33, 0.34"`. Atmos has the same [compat](https://github.com/CliMA/ClimaAtmos.jl/blob/main/Project.toml#L80C1-L80C25)

Reply:
Fixed.